File: 2025-08-20/squaresWithThree.js
function squaresWithThree(n) {
  let count = 0;
  for (let i = 1; i <= n; i ++) {
    let square = i ** 2;
    if (square.toString().includes('3'))
      count ++;
  }
  return count;
}

console.log(squaresWithThree(1))
console.log(squaresWithThree(10))
console.log(squaresWithThree(100))
console.log(squaresWithThree(1000))
console.log(squaresWithThree(10000))

File: 2025-08/2025-08-20/squares_with_three.py
def squares_with_three(n):
    count = 0;
    for i in range(1, n + 1):
        square = i ** 2;
        if '3' in str(square):
            count += 1
    return count

print(squares_with_three(1))
print(squares_with_three(10))
print(squares_with_three(100))
print(squares_with_three(1000))
print(squares_with_three(10000))